cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -120,4 +120,46 @@ Resources:
             {{> stateMachine}}
 
           - {}
-      RoleArn: !GetAtt [ StatesExecutionRole, Arn ]
\ No newline at end of file
+      RoleArn: !GetAtt [ StatesExecutionRole, Arn ]
+
+  NoStateMachineExecutionsAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      InsufficientDataActions:
+      - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:simple-dev
+      OKActions:
+      - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:simple-dev
+      AlarmName: !Sub No Recent State Machine Execution Attempts in ${Stage} (Monthly Contributions Sign-Ups)
+      AlarmDescription: The Monthly Contributions state machine has not been triggered for an extended period - has something broken?
+      MetricName: ExecutionsStarted
+      Namespace: AWS/States
+      Dimensions:
+        - Name: StateMachineArn
+          Value: !Ref MonthlyContributionsPROD
+      ComparisonOperator: LessThanThreshold
+      Threshold: 1
+      Period: 7200
```

Comment:
This might need some tuning, once I can work out exactly how AWS uses the period to determine whether there is insufficient data or not...